### ISSUE TITLE
2 menu plugins: 1 widget disappears when going fullscreen

### ISSUE THREAD

**Issue by phloxic** [flowplayer/flash#244]
Rare case, but not explicitly forbidden ;-)
See: https://flowplayer.org/forum/#!/setup_configuration:issue-with-variuos-plugins
I guess this is a tricky one.

**Comment by danrossi** [comment]
I've done this before but had to modify around the limitation to provide both a closed caption menu and alternate audio languages menu. from memory its because its trying to position and display to the menu button or something like that not sure.

**Comment by phloxic** [comment]
Yeah, one of 2 menu widgets just disappears in fullscreen - depending on how the rest of the controlbar widgets are configured there's an "empty space".
Of course you also cannot set a separate tooltip value for a second menu as far as I can see.
Personally I'm also open to document a restriction that only one menu plugin is allowed in a setup.

**Comment by danrossi** [comment]
ok just say the word it's trivial I know of the limitation and had to modify them a few times before.

**Comment by danrossi** [comment]
that is one of them. the widget name needs to be configurable its hardcoded as "menu"

**Comment by phloxic** [comment]
If it's trivial then by all means go for it, I'd say - @anssip ?
